demos/dev_mode_key_provisioning/src/aws_dev_mode_key_provisioning.c

```diff
@@ -865,6 +879,42 @@ static CK_RV prvGetProvisionedState( CK_SESSION_HANDLE xSession,
                                                 &pxProvisionedState->xClientCertificate );
     }
 
+    /* Check for a crypto element identifier. */
+    if( CKR_OK == xResult )
+    {
+        xResult = xGetSlotList( &pxSlotId, &ulSlotCount );
+    }
+
+    if( CKR_OK == xResult )
+    {
+        xResult = pxFunctionList->C_GetTokenInfo( pxSlotId[ 0 ], &xTokenInfo );
+    }
+
+    if( ( CKR_OK == xResult ) && ( '\0' != xTokenInfo.label[ 0 ] ) && ( ' ' != xTokenInfo.label[ 0 ] ) )
+    {
+        /* PKCS #11 requires that token info fields are padded out with space
+         * characters. However, a NULL terminated copy will be more useful to the
+         * caller. */
+        pcSpace = ( char * ) memchr( xTokenInfo.label, ' ', sizeof( xTokenInfo.label ) );
+    }
```

Comment:
What happens when this code runs on a non-ecc608a device?  Would like to verify that whatever token info SW ports return doesn't cause a string parsing issue.